include/tester/db/MessageLogger.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "LogSeverity.hpp"
+
+class MessageLogger
+{
+public:
+    MessageLogger()
+    {
+
+    }
+
+    MessageLogger(LogSeverity severity, const std::string& description):
```

Comment:
get `description` by value, and use `std::move` to initialize the relevant member

Reply:
done